src/processor.ts

```diff
@@ -373,6 +378,13 @@ processor.run(new TypeormDatabase({ isolationLevel: 'READ COMMITTED' }), async (
     exportBlockNumber = await offchainState.getExportBlockNumber()
   }
 
+  if (JOYSTREAM_USD_PRICE === null) {
+    await updateJoystreamPrice()
+    schedulePriceUpdate()
+      .then(() => undefined)
+      .catch(() => ctx.log.error('Fetching JOYSTREAM price failed'))
+  }
+
```

Comment:
Just noticed that you are updating the `JOYSTREAM_USD_PRICE` in the `orion_processor` process, however, the value is being used in the email-scheduler cron job, where the value will always be null.